### ISSUE TITLE
unc0ver 2.1.0. error loading respring 

### ISSUE THREAD

**Issue by repoxox** [pwn20wndstuff/Undecimus#221]
i'm use unc0ver lastest 2.1.0 , cydia substrate 0.9.7020. everything is good.
until today,I turned off the power and re-jailbreak it.but to the process ReSpring,it keeps loading forever.
and I temporarily fix it with unc0ver-2.1.1 pre.
and let me ask?why after every use of Rec0ver,Memory on the device increased to about 1gb.hope someone can tell me this problem!

**Comment by Cryptiiiic** [comment]
@repoxox post diagnostics from unc0ver settings tab.

**Comment by 0j0k3r** [comment]
I got the same issue, I got it since the 1st pre releases of 2.1.0. Im using the MP exploit, and the latest substrate update. When I reboot my device and try to jailbreak it again with u0, the jb process went fine (so far I can see it mostly it reboots at 37/40), but after the reboot the phone ends up in a respring loop, and I have to hard reset my phone. Most of the time it worked by the 2nd jb try, but yesterday I have to reJb my phone 6 times to get the jb fully working again. I added my logfile as .txt I hope thats okay to see whats eventualy goes wrong.



[Neues Textdokument.txt](https://github.com/pwn20wndstuff/Undecimus/files/2753442/Neues.Textdokument.txt)
